src/sentry/organizations/services/organization/impl.py

```diff
@@ -349,6 +352,31 @@ def update_flags(self, *, organization_id: int, flags: RpcOrganizationFlagsUpdat
             Organization.objects.filter(id=organization_id).update(flags=updates)
             Organization(id=organization_id).outbox_for_update().save()
 
+    def get_aggregate_project_flags(self, *, organization_id: int) -> RpcProjectFlags:
+        """We need to do some bitfield magic here to convert the aggregate flag into the correct format, because the
+        original class does not let us instantiate without being tied to the database/django:
+        1. Convert the integer into a binary representation
+        2. Pad the string with the number of leading zeros so the length of the binary representation lines up with the
+           number of bits of MAX_BIGINT / the BitField
+        3. Reverse the binary representation to correctly assign flags based on the order
+        4. Serialize as an RpcProjectFlags object
+        """
+        flag_keys = cast(list[str], Project.flags)
+
+        org = Organization.objects.filter(id=organization_id).get()
+        projects = org.project_set
+        if org.project_set.count() > 0:
```

Comment:
You could skip the read on `Organization` and use `Project` to create a queryset `Project.objects.filter(organization_id=organization_id)` would give you the organization's project list without needing to load the org record.